test/test_jit_py3.py

```diff
@@ -48,6 +48,21 @@ def foo(x) -> float:
 
         self.assertEqual(foo(torch.rand(3, 4)), 18.0)
 
+    @unittest.skipIf(sys.version_info[0] < 3 and sys.version_info[1] < 6, "dict not ordered")
+    def test_dict_preserves_order(self):
+        def dict_ordering():
+            a : Dict[int, int] = {}
+            for i in range(1000):
```

Comment:
This test can be more minimal by eliminating the computation of values at all:
```
for i in range(100):
  a[i] = 0
```
```

Reply:
i don't think it's a good practice to test keys and values with the same set of values

Reply:
You're only testing a property of keys here, not values.  Any test code having to do with values is irrelevant.